src/firmware/boards/arduino_uno/main.cc

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
another nit: would be good to put this in `estop_arduino_uno` folder

because we've ben doing

`thing_board` (`robot_stm32h7`)

Reply:
to clarify, you mean rename the folder `arduino_uno` to `estop_arduino_uno` ?

Reply:
yup!

Reply:
renamed file

Reply:
looks like it didn't change?